### DISCUSSION TITLE
收集柴犬色色卡

### DISCUSSION THREAD

**Discussion by cohhi** [steam-404/411message-6.0#4] [Q&A]
把你们的柴犬色色卡都交出来,就像这种,越多越好。
![dog1](https://user-images.githubusercontent.com/71576964/208830668-f8a677bc-3bf3-4c3b-b0f1-5affa8b52771.png)
![dog2](https://user-images.githubusercontent.com/71576964/208830685-180d3f19-aa76-4fc8-84ba-d53a94494202.png)

**Comment by li-nuonuo** [comment]
![-7bcd21b086bea574](https://user-images.githubusercontent.com/119656815/208833380-0b9c3502-1399-45d7-896e-e50704d1bcd1.gif)

**Comment by li-nuonuo** [comment]
![-16332a6d978243a5](https://user-images.githubusercontent.com/119656815/208833529-85a0187e-e5e7-4536-b6df-c3a38a49da85.gif)

**Comment by li-nuonuo** [comment]
![1686793328](https://user-images.githubusercontent.com/119656815/208833620-5f21dff1-b0e6-4131-924c-b6be2c84751f.jpeg)
